features/features.go

```diff
@@ -80,6 +80,11 @@ type Config struct {
 	// functionality (valid authz reuse) while letting us simplify our code by
 	// removing pending authz reuse.
 	NoPendingAuthzReuse bool
+
+	// MPICFullResults causes the VA to wait for all remote (MPIC) results, rather
```

Comment:
optional: Seems a little redundant when MPIC is in the name.

```suggestion
	// MPICFullResults causes the VA to wait for all remote results, rather
```